fv3gfs-util/fv3gfs/util/partitioner.py

```diff
@@ -716,31 +738,95 @@ def tile_extent_from_rank_metadata(
     Returns:
         tile_extent: the extent of one tile
     """
+    if edge_interior_ratio < 1.0:
+        print(f"{edge_interior_ratio}", flush=True)
+        raise NotImplementedError("Only equal sized subdomains are supported.")
```

Comment:
Usually better to include debugging information in the exception, print statements shouldn't exist in the final merged code.

```suggestion
        raise NotImplementedError(
            "Only equal sized subdomains are supported, was given "
            f"an edge_interior_ratio of {edge_interior_ratio}"
        )
```

Reply:
Has been changed. Not sure how that snuck by.